supervision/metrics/detection.py

```diff
@@ -0,0 +1,326 @@
+from dataclasses import dataclass
+from typing import Callable, List, Optional, Tuple
+import matplotlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import supervision as sv
```

Comment:
We can't use alias import - `import supervision as sv` here. It can cause problems after the package is built. Internally we use an explicit import - `from supervision.detection.core import Detections`.